src/main/java/io/xpring/xrpl/model/MemoField.java

```diff
@@ -0,0 +1,18 @@
+package io.xpring.xrpl.model;
+
+import org.immutables.value.Value;
+
+/**
+ * Provides a means of passing a string to a memo that allows for user
+ * specification as to whether or not the string is already a hex string.
+ */
+@Value.Immutable
+public interface MemoField {
+  static ImmutableMemoField.Builder builder() {
+    return ImmutableMemoField.builder();
+  }
+
+  String value();
```

Comment:
Would you mind doc'ing these? I know it's obvious to you since you've been reading the documentation but it may not be to someone without that context.

Reply:
doc'ed